packages/rule-html-checker/src/rule.ts

```diff
@@ -129,28 +129,39 @@ export default class HtmlCheckerRule implements IRule {
             await context.report(resource, null, `Couldn't get results from HTML checker for ${resource}. Error: ${error}`);
         };
 
-        const start = (data: FetchEnd) => {
-            const { response } = data;
-
+        const requestRetry = async (options, retries: number = 3) => {
             /*
              * HACK: Need to do a require here in order to be capable
              * of mocking when testing the rule and `import` doesn't
              * work here.
              */
-            const htmlChecker = require('html-validator');
+            const { requestAsync, delay } = require('sonarwhal/dist/src/lib/utils/misc');
```

Comment:
With TypeScript 2.8 we should be capable of doing dynamic imports. Can you please test it and see if it works?